estuary/models/koji.py

```diff
@@ -69,9 +70,22 @@ class ContainerKojiBuild(KojiBuild):
         '.freshmaker.FreshmakerEvent', 'TRIGGERED', cardinality=ZeroOrOne)
 
 
+class ModuleKojiBuild(KojiBuild):
+    """A Neo4j definition of a build that represents a module build in Koji."""
+
+    components = RelationshipTo('KojiBuild', 'ATTACHED')
```

Comment:
I think the relationship should actually be `CONTAINS`. Sorry I missed this.

Reply:
makes sense! I'll do it